src/components/Dashboard/TodoList.vue

```diff
@@ -1740,8 +1740,8 @@ export default class TodoList extends Vue {
 .todo-subtitle {
   color: $gray7;
   display: flex;
-  align-items: center;
   justify-content: flex-start;
+  flex-direction: column;
```

Comment:
Make sure this doesn't break any of the other item types.

Reply:
I checked this style using other filings, and they were looking fine.